### PR TITLE
Update to 2020 gradlerio and gradle 5.4.1

### PR DESCRIPTION
**Pull request by ThadHouse**

### CODE REVIEW

**Comment by AustinShalit** [comment]
Do you want to go direct to 5.4.2?

**Comment by ThadHouse** [comment]
We will likely be bumping to 6.0 when it releases, but right now everything is 5.4.1 so basically everything would need to be rebuilt

File: gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-gradleRioVersion: 2019.4.1
+gradleRioVersion: 2020.0.0-alpha-2
```
Comment (AustinShalit) [inline gradle.properties]: Are the line endings ok in this file?

Comment (ThadHouse) [reply gradle.properties]: Yes, this is fine, it really weird why its happening but it works just fine.

**Pull request merged**